# /** ------ 传感器定义和状态 ------ */
coord      sPos
coord      sOriginCoordSys
string     au1Name
uint32     u8HostId
uint32     u8Id
float32[]  au4FovHV
float32[]  au4ClipNF
float32[]  au4FovOffHV
uint8      u1Type                   
uint8      u1ModelType
float64	   u4IdealFreq
uint32	   u4ResL
uint32     u4ResV
float64	   u4SenSizeL
float64	   u4SenSizeW
float64	   u4LenFd
float64	   u4DistCenL
float64	   u4DistCenW
float64	   u4DistParK1
float64	   u4DistParK2
